_documentation/getting-started/subscribe.md

```diff
@@ -1,35 +1,12 @@
 ---
-title: Subscribing to a podcast
+title: documentation.categories.getting-started.subscribe
 layout: doc
 level: "2"
 group: "getting-started"
 icon: "fas fa-plus"
 order: "10"
 ---
 
-# Subscribing to a podcast
-The first thing you want to do after downloading a podcast app is to subscribe to a podcast.
+# {% t documentation.categories.{{ page.group }}.{{ page.slug }} %}
```

Comment:
Hmm. I would actually change this to explicitly mention the key (`documentation.categories.getting-started.subscribe`) instead of building it based on other values

Reply:
Yes, `# {% t {{ page.title }} %}` seems to work :)